frontend/src/app/components/Badge/index.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import React from "react";
+import { Badge as RadixBadge } from "@radix-ui/themes/components/badge";
+import { CrossCircledIcon } from "@radix-ui/react-icons";
+import IBadgeProps from "./badge.types";
+
+export default function Badge({
+  text,
+  bgColor = "gray",
+  icon,
+  handleRemoveClick,
+}: IBadgeProps) {
+  const isTransparent = bgColor === "transparent" ? true : false;
```

Comment:
```suggestion
  const isTransparent = bgColor === "transparent";
```

用 === 判斷之後就不需要自己轉換 true false 了～可以省一些字 😂